packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -422,7 +422,7 @@ describe('merge-on-green wrapper logic', () => {
 
         const [reducedPRs] = handler.maybeReducePRList(prs);
 
-        assert.strictEqual(reducedPRs.length, 25);
+        assert.ok(reducedPRs.length === 24 || reducedPRs.length === 25);
```

Comment:
We should be confident in this value.

Reply:
I think it's OK if we're not confident. It's produced by a random value, and the only requirement is that it has to be under 25 (the chunk size): https://github.com/googleapis/repo-automation-bots/blob/ce561fe11471717eaa93aea7cdf2fe2922140b23/packages/merge-on-green/src/merge-on-green.ts#L87. I've updated the test to reflect that.